lib/Conversion/TorchOnnxToTorch/Patterns.cpp

```diff
@@ -24,12 +25,25 @@ LogicalResult OnnxCustomOpConversionPattern::matchAndRewrite(
   auto foundIt = namedHandlers.find(op.getNameAttr());
   if (foundIt == namedHandlers.end())
     return failure();
+  // If the op has an individual version, use it.
+  int64_t opDomainVersion = domainVersion;
```

Comment:
Would it be possible to allow absence of the opset_version / domainVersion on the function as long as each op has its version?

Reply:
This is allowed:
* The check for per-function domain version [is gone](https://github.com/Xilinx/torch-mlir/pull/183/files#diff-7fee8580cc0c5225a47c00ba1256929e524858d64ebf5758795e70b369b0b981L48), and by default the opset version is 0,
* If the op has a version, it overrides the per-function domain version,
* Otherwise, the version is 0, which will match no pattern.

Reply:
Looks good, thanks!